linera-views/src/test_utils/performance.rs

```diff
@@ -0,0 +1,273 @@
+// Copyright (c) Zefchain Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use std::{
+    fmt::Debug,
+    time::{Duration, Instant},
+};
+
+use crate::{
+    batch::Batch,
+    store::LocalKeyValueStore,
+    test_utils::{add_prefix, get_random_key_values2},
+};
+
+// We generate about 2000 keys of length 11 with a key of length 10000
+// The keys are of the form 0,x_1, ..., x_n with 0 <= x_i < 4 and n=10.
+
+/// A value to use for the keys
+const PREFIX: &[u8] = &[0];
+
+/// A value to use for the keys
+const PREFIX_SEARCH: &[u8] = &[0, 0];
+
+/// The number of keys
+const NUM_ENTRIES: usize = 200;
+
+/// The number of inserted keys
+const NUM_INSERT: usize = 70;
+
+/// The length of the keys
+const LEN_KEY: usize = 10;
+
+/// The length of the values
+const LEN_VALUE: usize = 10000;
+
+async fn clear_store<S: LocalKeyValueStore>(store: &S) {
+    let mut batch = Batch::new();
+    batch.delete_key_prefix(PREFIX.to_vec());
+    store.write_batch(batch).await.unwrap();
+}
+
+/// Compute benchmark of the `contains_key` operation.
```

Comment:
```suggestion
/// Computes benchmark of the `contains_key` operation.
```

(Also below.)

Or maybe:

```suggestion
/// Benchmarks the `contains_key` operation.
```

Reply:
Ok for "Benchmarks".